### PR TITLE
Add objectives

### PR DESCRIPTION
**Pull request by getaaron**
I added Bloom-style objectives and made some other minor changes to fit more closely with the style of the other courses. Once this looks good I'll update the README

### CODE REVIEW

**Comment by getaaron** [comment]
I'm still pretty new to Angular so please let me know if I explained anything incorrectly

File: workshop/exercise_3.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
-# Angular Controllers
-
 > Description: Set up your first Angular controller
 > Command: `mocha test/exercise_3.js --timeout 5000 --reporter test/log_reporter.js`
 
-Usually we won't be interpreting just basic arithmetic in the HTML, but actual JavaScript objects containing data. Let's interpret a single task from our todo list with:
+Although AngularJS can interpret arithmetic in HTML, it's typically used to display JavaScript objects containing data.
```
Comment (mikesjewett) [inline workshop/exercise_3.md]: > can interpret arithmetic

- [ ] sounds awkward to me. Is it grammatically correct?

File: workshop/exercise_6.md [hunk 2]
```diff
@@ -72,4 +85,18 @@ myApp.controller("TaskController", function($scope){
...
+
+In the controller…
+
+- Initialize an empty object called `newTask` on the scope
+- Add the `addTask` function to the scope
+
+In your HTML…
+
+- Add the `ng-model` attribute to bind the text input field to `newTask.name`
+- Add the `ng-click` attribute so the submit button calls the `addTask` function
```
Comment (mikesjewett) [inline workshop/exercise_6.md]: - [ ] We should add some sort of conclusion statement here, perhaps?

**Comment by mikesjewett** [comment]
Looks awesome @getaaron , just a couple little comments.

**Pull request merged**